package.json

```
@@ -7,15 +7,15 @@
     "i18next-scanner": "^3.1.0"
   },
   "scripts": {
-    "postinstall": "EXECUTE_PARALLEL=true node workspace-run.js build:lib",
+    "postinstall": "cross-env EXECUTE_PARALLEL=true node workspace-run.js build:lib",
     "pre-release": "node workspace-run.js pre-release",
     "start": "yarn workspace @talend/ui-playground run start",
     "info": "yarn workspaces --silent info > info.json",
     "release": "yarn changeset publish",
     "lint:es": "cross-env WORKSPACE_RUN_FAIL=no-bail node workspace-run.js lint:es",
     "lint:style": "cross-env WORKSPACE_RUN_FAIL=no-bail node workspace-run.js lint:style",
     "build": "yarn workspaces run build",
-    "test": "EXECUTE_PARALLEL=true cross-env TZ=Europe/Paris node workspace-run.js test --silent",
+    "test": "cross-env EXECUTE_PARALLEL=true cross-env TZ=Europe/Paris node workspace-run.js test --silent",
```

it seems that you only need to put it once by command even with two variables

---

regarding their documentation
`{
  "scripts": {
    "build": "cross-env FIRST_ENV=one SECOND_ENV=two node ./my-program"
  }
}`

so 
```suggestion
    "test": "cross-env EXECUTE_PARALLEL=true TZ=Europe/Paris node workspace-run.js test --silent",
```